static/js/app.js

```diff
@@ -1225,6 +1251,38 @@ function bindTableHeaderMenu() {
   });
 }
 
+function getColumnsFromResults(){
+  let columns = [];
+    $("#results_header th").each(function(){
+    columns.push(this.innerText);
+    })
+  return columns;
+}
+
+function getValuesFromContext(context){
```

Comment:
Indentation on the following code is all messed up, please fix. Also remove the commented out line.